### ISSUE TITLE
[Feature Request] Add a player_name param to `monitor_touch` event

### ISSUE THREAD

**Issue by zyxkad** [SirEndii/AdvancedPeripherals#442]
### Describe your idea

I want to make monitors more interactive with players

For example, when player click the monitor, I can use chatBox send the player a link to open, or create a game allow two players play at the same time (but not use two monitors or computers).

### Additional context

probably mouse events can also have this param

**Comment by SirEndii** [comment]
Yeah, but I don't get around adding this for 0.8
Someone could make a pr - but AP is currently a bit paused for me

**Comment by zyxkad** [comment]
Nvm, if I have time I can try to implement it (probably not, I don't know much about forge api).

**Comment by zyxkad** [comment]
I check the CC:T codes, if I want to impl that, I should use mixin, because the signature of the `monitorTouched` method don't have player param. I'm not sure if you would like to do that.
```java
    void monitorTouched(float xPos, float yPos, float zPos)
```

**Comment by SirEndii** [comment]
Yeah. That's a reason why I do bot want to add features to existing blocks from CC
